### PR TITLE
Garl hotfix

### PR DESCRIPTION
**Pull request by ItsSudip**
## Description of the change

> Description here

## Type of change
- [ ] Bug fix (non-breaking change that fixes an issue)
- [ ] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Related issues

> Fix [#1]() 

### Development

- [ ] Lint rules pass locally
- [ ] The code changed/added as part of this pull request has been covered with tests
- [ ] All tests related to the changed code pass in development

### Code review 

- [ ]  This pull request has a descriptive title and information useful to a reviewer. There may be a screenshot or screencast attached
- [ ] "Ready for review" label attached to the PR and reviewers mentioned in a comment
- [ ] Changes have been reviewed by at least one other engineer
- [ ] Issue from task tracker has a link to this pull request

### CODE REVIEW

**Review by saikumarrs — APPROVED** [review]

File: v0/destinations/google_adwords_remarketing_lists/transform.js [hunk 1]
```diff
@@ -213,7 +215,7 @@ const processEvent = async (metadata, message, destination) => {
 
     if (!Object.keys(createdPayload).length) {
       throw new CustomError(
-        "[Google_adwords_remarketing_list]:: add or remove property is not present inside listData. Aborting message.",
+        "[Google_adwords_remarketing_list]:: add or remove property is not present inside listData or there is not any attribute inside add or remove matching with the schemafields. Aborting message.",
```
Comment (saikumarrs) [inline v0/destinations/google_adwords_remarketing_lists/transform.js:218]:
```suggestion
        "[Google_adwords_remarketing_list]:: Neither 'add' nor 'remove' property is present inside 'listData' or there are no attributes inside 'add' or 'remove' properties matching with the schema fields. Aborting message.",
```

File: v0/destinations/google_adwords_remarketing_lists/transform.js [hunk 2]
```diff
@@ -145,6 +140,13 @@ const createPayload = (message, destination) => {
         listData[key],
         destination
       );
+      if (userIdentifiersList.length === 0) {
+        logger.info(
+          `[Google_adwords_remarketing_list]:: No attributes are there in the ${key}.`
```
Comment (saikumarrs) [inline v0/destinations/google_adwords_remarketing_lists/transform.js:145]:
```suggestion
          `[Google_adwords_remarketing_list]:: No attributes are present in the '${key}' property.`
```

**Comment by utsabc** [comment]
Why AC fix is present in this branch?

**Comment by ItsSudip** [comment]
This came because I was working on AC and somehow it stayed in the local branch. I tried to pull from the master but it didn't work. After merging the AC PR, I think we can pull once again and remove the conflicts.

**Comment by utsabc** [comment]
> This came because I was working on AC and somehow it stayed in the local branch. I tried to pull from the master but it didn't work. After merging the AC PR, I think we can pull once again and remove the conflicts.

AC changes are merged

**Review by utsabc — APPROVED** [review]

**Review by saikumarrs — APPROVED** [review]

**Pull request merged**